db/schema.rb

```diff
@@ -13,6 +13,7 @@
 ActiveRecord::Schema.define(version: 2019_02_02_172750) do
 
   # These are extensions that must be enabled in order to support this database
+  enable_extension "hstore"
```

Comment:
We are not using this.
Please do rails db:drop create migrate

Reply:
Done